src/pyird/spec/continuum.py

```diff
@@ -87,10 +87,20 @@ def f(scale):
         df_tmp = pd.DataFrame(data,columns=['wav','order','flux','flat','continuum'])
         df_continuum = pd.concat([df_continuum,df_tmp],ignore_index=True)
     df_continuum['nflux'] = df_continuum['flux']/df_continuum['continuum']
-    df_continuum['nflat'] = df_continuum['flat']/df_continuum['continuum']
+    ###############
+    if (wdata['wav']>1420).any():
```

Comment:
Avoid hard coding of numbers. Define the value somewhere separately and refer to that. I think @YuiKasagi can tell where the quantities should be defined :)
```python
wav_boundary_yj_and_h = 1420.0 
```
same as `gain` and `uncertain_gain`

Reply:
@YuiKasagi Could you give some advice on this issue? thank you

Reply:
I think that `wav_boundary_yj_and_h` should be defined in the comb_norm() and add the parameter to make_blaze() and normalize(). It's like the following.

```
def comb_norm(wfile,flatfile,LFC_path):
...
    wav_boundary_yj_and_h = 1420.0
    df_continuum = make_blaze(wdata,flat,RON,wav_boundary_yj_and_h)
    df_interp = normalize(df_continuum,RON,wav_boudary_yj_and_h)
    return df_continuum, df_interp
```

(It might be better to create a file which is summarized the instrument characteristics. (← just a memo for the next update plan))